app/src/main/java/org/fossasia/openevent/general/event/EventsFragment.kt

```diff
@@ -44,6 +44,7 @@ class EventsFragment : Fragment() {
                 val bundle = Bundle()
                 bundle.putLong(fragment.EVENT_ID, eventID)
                 fragment.arguments = bundle
+                eventsViewModel.loadEventsWithAnimation = false
```

Comment:
Rather than explaining that in the comment, you removed the comment altogether

Reply:
Meaningful comment added

Reply:
Add the comment above this statement

Reply:
PR updated